search_events_app/views.py

```diff
@@ -1,12 +1,12 @@
 from django.shortcuts import render
 from django.views.generic.list import ListView
 
-from .services.dummy_api import DummyApi
+from .services.api_service import get_events
 
 
 # Create your views here.
 class EventListView(ListView):
     template_name = 'event_list.html'
 
     def get_queryset(self):
-        return DummyApi().get()
+        return get_events
```

Comment:
isn't `get_events` a method?
is that what you want to return?

Reply:
Yes, Done in the new commit!